specification/azurefleet/resource-manager/Microsoft.AzureFleet/preview/2023-11-01-preview/azurefleet.json

```diff
@@ -486,42 +486,6 @@
     }
   },
   "definitions": {
-    "Azure.ResourceManager.CommonTypes.ManagedServiceIdentityType": {
```

Comment:
those azure fleet change I think is the only think marked as a breaking change.

Reply:
reason for it, arm library didn't used to be able to define the reference for enum/union types so those were included in teh spec. Now we can use the common types